### ISSUE TITLE
Auto-publish to NuGet

### ISSUE THREAD

**Issue by Smaug123** [G-Research/fsharp-analyzers#73]
Is there a reason we're currently leaving changes in "unreleased" state at all? (For example, https://github.com/G-Research/fsharp-analyzers/actions/runs/7726152791/job/21061944118 on commit 10808cb33f001365cc6c189a8099f164810cc3e9 didn't release a new version because it wanted to publish to the same 0.8.0 version that had already been published earlier.) The "underscore in named placeholders" bugfix isn't *strictly* a prerequisite for me testing the analysers on my team's monorepo, but it's currently quite a noisy false positive.

**Comment by nojaf** [comment]
There is no good reason in this particular case.
We will ship a new version once @dawedawe has updated to the latest analyzer SDK.
Sorry, for not shipping this earlier.

**Comment by dawedawe** [comment]
Yes, sorry. I didn't realize that this was anoying to you and a new release would have made sense.

**Comment by nojaf** [comment]
I think we are better at this these days.
The process is automated, the human still needs to put in the entry in the changelog.
I'd say this is fine for now.
